addons/core/.eslintignore

```diff
@@ -1,25 +1,17 @@
 # unconventional js
 /blueprints/*/files/
-/vendor/
 
 # compiled output
+/declarations/
 /dist/
-/tmp/
 
 # dependencies
-/bower_components/
 /node_modules/
```

Comment:
i think this `node_modules` specification might not needed anymore since according to the docs node_modules are [ignored by default](https://eslint.org/docs/v8.x/use/configure/ignore#:~:text=In%20addition%20to%20any%20patterns%20in%20the%20.eslintignore%20file%2C%20ESLint%20always%20follows%20a%20couple%20of%20implicit%20ignore%20rules%20even%20if%20the%20%2D%2Dno%2Dignore%20flag%20is%20passed.%20The%20implicit%20rules%20are%20as%20follows%3A). Same thing goes for the `.prettierignore` file ([docs here](https://prettier.io/docs/cli#:~:text=glob%20module.-,Prettier%20CLI%20will%20ignore%20files%20located%20in%20node_modules%20directory,-.%20To%20opt%20out)).

Reply:
Ahhh I was doubting about this one tbh. I understood deleting `bower_components` made sense since bower is long gone, but reading the ignored by default link you provided makes sense to delete it! thanks!